basicsynbio/functions/pdf_instructions.py

```diff
@@ -0,0 +1,409 @@
+    )
+    elems.append(Spacer(1, 0.4 * cm))
+    elems.append(
+        Table(
+            PROCESSED_MASTER_MIX_BASIC_REACTION,
+            colWidths=[7 * cm, 7 * cm],
+            style=style_no_split,
+        )
+    )
+    elems.append(
+        Paragraph(
+            "For each Clip reaction, setup 1 PCR tube with 30 μl total volume: ",
+            styles["BodyText"],
+        )
+    )
+    elems.append(
+        Paragraph(
+            "Dispense 20 uL master mix, 1 μl of each prefix and suffix Linker, 1 μl of part (or more depending on concentration) into a PCR tube and make up to 30 μl with water.",
+            styles["BodyText"],
+        )
+    )
+    elems.append(Spacer(1, 0.4 * cm))
+    elems.append(
+        Paragraph(
+            "We recommend the above up each clip can be used in up to 28 assemblies.",
+            styles["BodyText"],
+        )
+    )
+    elems.append(Spacer(1, 0.4 * cm))
+    elems.append(
+        Paragraph(
+            "After mixing, tubes are placed in a PCR machine running the following programme: ",
+            styles["BodyText"],
+        )
+    )
+    elems.append(Spacer(1, 0.4 * cm))
+    elems.append(
+        Table(
+            PROCESSED_TEMPERATURE_PROFILE_1,
+            colWidths=[4 * cm, 4 * cm, 4 * cm],
+            style=style_joint_cell,
+        )
+    )
+    ## Magbead Purification 2
+    elems.append(Paragraph("Magbead purification", styles["Heading2"]))
+    elems.append(
+        Paragraph(
+            "Prepare fresh 70% EtOH (0.5 ml per BASIC reaction) and bring magnetic beads (AmpureXP or Ampliclean) stored at 4°C back into homogeneous mix by shaking thoroughly. ",
+            styles["BodyText"],
+        )
+    )
+    elems.append(
+        Paragraph(
+            "We recommend using a 96 well Falcon plate (Falcon 351177) in combination with an Ambion magnetic plate (AM10050) for quick magbead immobilisation and easy pipetting access. ",
+            styles["BodyText"],
+        )
+    )
+    elems.append(
+        ListFlowable(
+            [
+                Paragraph(
+                    "Add 54 μl of magnetic beads into 96 well Falcon plate (one well per BASIC reaction) and add the 30 μl BASIC linker ligation from the PCR machine step, mix by pipetting 10 times. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Wait 5min to allow DNA binding to magbeads. ",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Place Falcon plate on magnetic stand and wait for rings to form and solution to clear.",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Remove the solution with a 200 μl pipette tip from the centre of each well.",
+                    styles["BodyText"],
+                ),
+                Paragraph(
+                    "Add 190 μl 70'%' EtOH to each well and wait 30 s. ",
```

Comment:
Can we remove quotation marks around the % sign? I think there's a few other instances of this.